updateMojang.py

```diff
@@ -139,25 +136,6 @@ def main():
             else:
                 print("Already have experiment " + version.id)
 
-    static_old_snapshots_path = os.path.join(STATIC_DIR, STATIC_OLD_SNAPSHOTS_FILE)
-
-    # deal with old snapshots
-    if os.path.exists(static_old_snapshots_path):
-        old_snapshots = OldSnapshotIndexWrap(
-            OldSnapshotIndex.parse_file(static_old_snapshots_path)
-        )
-        old_snapshots_ids = set(old_snapshots.versions.keys())
-
-        for x in old_snapshots_ids:
-            version = old_snapshots.versions[x]
-            old_snapshots_path = os.path.join(UPSTREAM_DIR, VERSIONS_DIR, f"{x}.json")
-
-            print("Updating old snapshot " + version.id)
-            if not os.path.isfile(old_snapshots_path):
-                fetch_modified_version(old_snapshots_path, version)
-            else:
-                print("Already have old snapshot " + version.id)
-
```

Comment:
I think we would still want this: just mapping the old_snapshots to the `snapshot` type should suffice

Reply:
And this has returned on request.